Cairo Desktop/CairoDesktop.Configuration/Properties/Settings.settings

```diff
@@ -98,5 +98,8 @@
     <Setting Name="FullWidthTaskBar" Type="System.Boolean" Scope="User">
       <Value Profile="(Default)">False</Value>
     </Setting>
+    <Setting Name="RunAtLogOn" Type="System.Boolean" Scope="User">
```

Comment:
This is the last place the old setting is referenced. You should be able to remove this by going to the CairoDesktop.Configuration properties, then choosing Settings, and removing this setting.